esp/platform/espcfg.ipp

```diff
@@ -198,13 +198,16 @@ public:
     void loadProtocols();
     void loadServices();
     void loadBindings();
+    void startEsdlMonitor();
 
     void loadAll()
     {
         DBGLOG("loadServices");
         loadServices();
         loadProtocols();
         loadBindings();
+        if(useDali)
```

Comment:
as a heads up, the dali detach feature will be merged soon, you'll have to be aware of the current attach mode